dashboard/app/helpers/script_levels_helper.rb

```diff
@@ -68,4 +72,14 @@ def tracking_pixel_url(script)
       CDO.code_org_url "/api/hour/begin_#{script.name}.png"
     end
   end
+
+  def can_show_ta_scores_alert?(lesson)
+    puts "checking can show ta scores alert for lesson #{lesson.id}"
+    puts "learning goal: #{LearningGoalTeacherEvaluation.where(teacher_id: current_user.id)}"
+    puts "goal exists: #{LearningGoalTeacherEvaluation.where(teacher_id: current_user.id).where.not(understanding: nil).exists?}"
```

Comment:
Intended to leave in?  Just noticed them on local dashboard.  :)

Reply:
Oh whoops good catch! Will remove these asap!